src/Modal.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace atk4\ui;
+
+class Modal extends View
```

Comment:
Just wondering, perhaps Modal should extend Virtual Page?

``` php
$modal = $layout->add(['Modal', 'modal title go here']);

$layout->add('Button')->on('click', $modal->js()->modal('show'));

$modal->add('LoremIpsum'); // already work fine since it's virtualpage.